portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.SortDirection;	
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comment")
+public class CommentServlet extends HttpServlet {
+
+  /* The text param is a string property identifier for setting and retrieving
+   the body of textual content stored for a comment made by the user. */
+  private static final String TEXT_PARAM = "text";
+
+  /* The textin param is a unique string identifier for retrieving the comment text from a user through the form input in POST requests */
+  private static final String TEXTIN_PARAM = "text-input";
+
+  /* The name param is string property identifier for setting and retrieving the name of the user associated with a submitted comment */
+  private static final String NAME_PARAM = "name";
```

Comment:
These two comment lines get pretty long, could you split into separate lines like you did for other comments?